libethash-cl/CMakeLists.txt

```diff
@@ -17,3 +17,6 @@ TARGET_LINK_LIBRARIES(${EXECUTABLE} ${OpenCL_LIBRARIES} ethash)
 install( TARGETS ${EXECUTABLE} RUNTIME DESTINATION bin ARCHIVE DESTINATION lib LIBRARY DESTINATION lib )
 install( FILES ${HEADERS} DESTINATION include/${EXECUTABLE} )
 
+if (NOT ETHASHCL)
+	install( TARGETS ${LIBRARY} RUNTIME DESTINATION bin ARCHIVE DESTINATION lib LIBRARY DESTINATION lib )
```

Comment:
That's interesting ...

This was in cpp-ethereum until now and was committed by: #1756.
Due to it being made in cpp-ethereum and never mirrored in the actual ethash repository it got lost during the subtreee merging from #1816 

I just noticed it now and wanted to put it back. What would the proper way to do it be then? And are you sure that it's not needed? I remember this solving some build problems before this got committed.